CMakeLists.txt

```diff
@@ -53,19 +53,19 @@ include_directories (${CMAKE_CURRENT_SOURCE_DIR}/src)
 include_directories (${CMAKE_CURRENT_BINARY_DIR}/src)
 
 # testing & coverage
-if (${enable_tests})
+if (${ENABLE_TESTS})
   set (GTEST_SOURCE_DIR /usr/src/gtest/src)
```

Comment:
We need to have a test here:

```
if (EXISTS /usr/src/googletest/src)
    set (GTEST_SOURCE_DIR /usr/src/googletest/src)
else
    set (GTEST_SOURCE_DIR /usr/src/gtest/src)
endif ()
```

Reply:
Amended via extra commit. Please revisit.